app/services/response_file_builder/build_response_pdf_file.rb

```diff
@@ -56,127 +63,153 @@ def pdf_fields
-      result['2.4 phone number'] = respondent.address_telephone_number
-      result['2.4 mobile number'] = respondent.alt_phone_number
-      result['2.5'] = respondent.contact_preference || 'Off'
-      result['2.6 email address'] = respondent.email_address
-      result['2.6 fax number'] = respondent.fax_number
-      result['2.7'] = respondent.organisation_employ_gb
-      result['2.8'] = respondent.organisation_more_than_one_site ? 'yes' : 'no'
-      result['2.9'] = respondent.employment_at_site_number
+      apply_field result, respondent.name, :respondent, :name
+      apply_field result, respondent.contact, :respondent, :contact
+      apply_field result, address.building, :respondent, :address, :building
+      apply_field result, address.street, :respondent, :address, :street
+      apply_field result, address.locality, :respondent, :address, :locality
+      apply_field result, address.county, :respondent, :address, :county
+      apply_field result, address.post_code.tr(' ', ''), :respondent, :address, :post_code
+      apply_field result, respondent.dx_number, :respondent, :address_dx_number
+      apply_field result, respondent.address_telephone_number, :respondent, :phone_number
+      apply_field result, respondent.alt_phone_number, :respondent, :mobile_number
+      apply_field result, respondent.contact_preference, :respondent, :contact_preference
+      apply_field result, respondent.email_address, :respondent, :email_address
+      apply_field result, respondent.fax_number, :respondent, :fax_number
+      apply_field result, respondent.organisation_employ_gb, :respondent, :employ_gb
+      apply_field result, respondent.organisation_more_than_one_site, :respondent, :multi_site_gb
+      apply_field result, respondent.employment_at_site_number, :respondent, :employment_at_site_number
     end
 
     def apply_acas_pdf_fields(result)
-      result['new 3.1'] = response.agree_with_early_conciliation_details ? 'Yes' : 'No'
-      result['new 3.1 If no, please explain why'] = response.disagree_conciliation_reason
+      apply_field result, response.agree_with_early_conciliation_details?, :acas, :agree
+      apply_field result, response.disagree_conciliation_reason, :acas, :disagree_explanation
     end
 
     def apply_employment_details_pdf_fields(result)
-      result['3.1'] = response.agree_with_employment_dates ? 'yes' : 'no'
-      result['3.1 employment started'] = response.employment_start.try(:strftime, '%d/%m/%Y')
-      result['3.1 employment end'] = response.employment_end.try(:strftime, '%d/%m/%Y')
-      result['3.1 disagree'] = response.disagree_employment
-      result['3.2'] = response.continued_employment ? 'yes' : 'no'
-      result['3.3'] = tri_state_value_for(response.agree_with_claimants_description_of_job_or_title)
-      result['3.3 if no'] = response.disagree_claimants_job_or_title ? 'yes' : 'no'
+      apply_field result, response.agree_with_employment_dates, :employment_details, :agree_with_dates
```

Comment:
Boolean or tri state fields are now normalized as part of the lookup from the yaml file - so they return true/false for boolean and optionally nil for tri state (when nothing is selected)
This is because the welsh pdf template did not use the same values of "yes", "no" and "Off"